Cargo.toml

```diff
@@ -13,3 +13,8 @@ default-members = [
 
 [profile.release]
 opt-level = "s"
+
+[patch.crates-io]
+# Needed to make coverage work on MacOS (https://github.com/servo/core-foundation-rs/pull/357)
+core-foundation = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
+core-foundation-sys = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
```

Comment:
This is a bit scary because it could silently break something or we might not notice that is no longer necessary. On the other hand if I understand it correctly this library is only used on macos anyway.

Reply:
I think, because we commit our lock file (which includes git revision and versions for every dependency), I don't think it can really break "silently". We can, however, run into issues if we try to update dependencies for example.

Reply:
If there is a way to only patch the macOS target we could do that, but I wasn't able to find a way.